.github/workflows/artifact_testing.yml

```diff
@@ -0,0 +1,53 @@
+name: check_tarantool_integration
```

Comment:
Let's compare `workflow_name / job_name` lengths.

`fast_testing / linux (1.10)` — 27 symbols, `check_tarantool_integration / run_tests` — 39 symbols. When we open a list of checks on a commit, the name is cut to 20 symbols. So Ideally we should keep those names short.

![image](https://user-images.githubusercontent.com/92749/136478810-3562f52b-c9bb-4385-b12b-3cda835c244a.png)

I would also keep the name consistent with the workflow file name.

Reply:
Now the workflow file is named differently. Please check the latest changes.